samples/notebooks/M-Admin/extra_user_apps/apps/codeserver_proxy/install.sh

```diff
@@ -20,4 +20,10 @@ curl -fsSL https://code-server.dev/install.sh | sh
 code-server --install-extension ms-python.python --force
 code-server --install-extension ms-toolsai.jupyter --force
 code-server --install-extension mtxr.sqltools --force
-pip install .
\ No newline at end of file
+code-server --install-extension extension-vsix/mtxr.sqltools-driver-pg-0.2.0.vsix --force
```

Comment:
here you can pull the binaries if need to , from the web.

Reply:
I was not able to pull the install for any drivers while on the console or during the build.  The vsix files are definitions, not binaries.  I see no other way to get the drivers installed than to store the vsix files of the binaries that we cannot install via pulling from the web in automation.

Reply:
I will try one more thing....from the VS Marketplace, the description is "This package is part of vscode-sqltools extension."-- perhaps mtxr.vscode-sqltools.sqltools-driver-pg is the full name of the extension??   If it fails, I suggest leaving the vsix files in the repo.

Reply:
Working on a resolution to embed the redshift driver into the image

Reply:
All binaries removed